src/Host.cpp

```diff
@@ -222,19 +222,86 @@ Host::~Host()
 void Host::saveModules(int sync, bool backup)
 {
     QMapIterator<QString, QStringList> it(modulesToWrite);
-    QStringList modulesToSync;
-    QString dirName = mudlet::getMudletPath(mudlet::moduleBackupsPath);
-    QDir savePath = QDir(dirName);
-    if (!savePath.exists()) {
-        savePath.mkpath(dirName);
+    mModulesToSync.clear();
+    QString savePath = mudlet::getMudletPath(mudlet::moduleBackupsPath);
+    auto savePathDir = QDir(savePath);
+    if (!savePathDir.exists()) {
+        savePathDir.mkpath(savePath);
     }
     while (it.hasNext()) {
         it.next();
         QStringList entry = it.value();
+        QString moduleName = it.key();
         QString filename_xml = entry[0];
-        // CHECKME: Consider changing datetime spec to more "sortable" "yyyy-MM-dd#HH-mm-ss" (1 of 6)
-        QString time = QDateTime::currentDateTime().toString("dd-MM-yyyy#hh-mm-ss");
+
+        if (filename_xml.endsWith(QStringLiteral("mpackage"), Qt::CaseInsensitive) || filename_xml.endsWith(QStringLiteral("zip"), Qt::CaseInsensitive)) {
+        } else if (backup) {
```

Comment:
Isn't there something missing from the first `if` branch above this line - otherwise why is it there? :confused:

Reply:
Will fix.